lib/senkyoshi/models/content.rb

```diff
@@ -30,12 +30,17 @@ class Content < Resource
       "Senkyoshi::Quiz" => "Quizzes::Quiz",
     }.freeze
 
-    attr_accessor(:title, :body, :id, :files, :url)
+    attr_accessor(:title, :body, :files, :url)
     attr_reader(:extendeddata)
 
+    def initialize(resource_id = nil)
+      super(resource_id)
+    end
```

Comment:
What is the purpose of this `initialize` method? It's only calling `super`. Could we remove this method and just let it use the inherited `initialize` method?